.github/workflows/trivy_scan.yml

```diff
@@ -5,6 +5,13 @@ on:
     # every day at 10am
     - cron: '0 10 * * *'
   workflow_dispatch:
+    inputs:
+      # this is useful for testing an arbitrary released version of Antrea
+      # by default, we will use the latest release (obtained using the Github API)
+      antrea-version:
+        description: 'The released Antrea version to scan'
+        required: false
+        default: ''
```

Comment:
Can this be removed if the default one is actually configured by `curl`? I feel it might be confused with above comments.